guides/common/modules/ref_system-requirements-quickstart.adoc

```diff
@@ -26,3 +30,24 @@ ifdef::foreman-el,katello,orcharhino[]
 endif::[]
 * Install {ProductName} on a freshly provisioned system that serves no other function except to run {ProductName}.
 * Ensure the system has at least {project-minimum-memory} of memory.
```

Comment:
@evgeni Should we also mention minimum CPUs? Are users likely to hit the same issue I did with a single CPU?

Reply:
Probably, yes. Right now the only boundary we have is https://github.com/theforeman/foremanctl/blob/master/src/vars/tuning/development.yaml, which is 2 CPU, 4GB RAM, which is nowhere you would run even a proof-of-concept on.

In our test env we use 4CPU, 10GB RAM, which seems OK. We should update the vars file accordingly too.